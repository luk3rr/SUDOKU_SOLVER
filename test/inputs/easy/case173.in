000000630 400800000 700009000 008040700 090120000 000030820 079000503 801000000 002070010 
