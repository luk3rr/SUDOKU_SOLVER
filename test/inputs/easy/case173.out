Solving the following grid:
0 0 0 | 0 0 0 | 6 3 0 
4 0 0 | 8 0 0 | 0 0 0 
7 0 0 | 0 0 9 | 0 0 0 
------+-------+------
0 0 8 | 0 4 0 | 7 0 0 
0 9 0 | 1 2 0 | 0 0 0 
0 0 0 | 0 3 0 | 8 2 0 
------+-------+------
0 7 9 | 0 0 0 | 5 0 3 
8 0 1 | 0 0 0 | 0 0 0 
0 0 2 | 0 7 0 | 0 1 0 


Solution found :')

9 8 5 | 4 1 2 | 6 3 7 
4 2 3 | 8 6 7 | 1 9 5 
7 1 6 | 3 5 9 | 4 8 2 
------+-------+------
2 3 8 | 9 4 6 | 7 5 1 
5 9 7 | 1 2 8 | 3 6 4 
1 6 4 | 7 3 5 | 8 2 9 
------+-------+------
6 7 9 | 2 8 1 | 5 4 3 
8 4 1 | 5 9 3 | 2 7 6 
3 5 2 | 6 7 4 | 9 1 8 

Algorithm: BFS
Total time: 113 ms
Total expanded states: 16795
